### ISSUE TITLE
Insufficient Contrast Ratio between Background and Foreground on "Create new tag" page

### ISSUE THREAD

**Issue by maggiehua01** [CMU-313/Mayan-EDMS#22]
The accessibility score is currently 83 and gives the warning "Background and foreground colors do not have a sufficient contrast ratio." Low-contrast text is difficult or impossible for many users to read.
